<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinTranslatorMainWidget</class>
 <widget class="QWidget" name="FinTranslatorMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FinTranslatorMainWidget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="topMenuLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="tabBarLayout">
        <property name="spacing">
         <number>40</number>
        </property>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QPushButton" name="textTabButton">
          <property name="text">
           <string>텍스트</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignLeft">
         <widget class="QPushButton" name="docTabButton">
          <property name="text">
           <string>문서</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rightAlignLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QPushButton" name="settingsButton">
          <property name="text">
           <string>설정</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="statusLayout"/>
    </item>
    <item row="1" column="0">
     <widget class="QStackedWidget" name="mainStackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
